_rules/Kj3fQk.md

```diff
@@ -0,0 +1,151 @@
+---
+name: CSS orientation lock via media queries
+
+test_type: atomic
+
+description: |
+  This rule checks that page content is not locked to any specific display orientation, using CSS media queries.
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any `element` that is [visible](#visible) and [included in the accessibility tree](#included-in-the-accessibility-tree) of the [page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s).
+
+### Expectation
+
+Each target element does not have any `css` applied, that locks the page content to a preferred orientation.
```

Comment:
"preferred orientation" is the term normally used about what the user prefers. 

I would suggest using "specific orientation" here instead.